spec/controllers/hub/clients_controller_spec.rb

```diff
@@ -614,11 +614,25 @@
           }
         end
 
-        it "shows the full amount of tax returns" do
-          get :index, params: params
+        context "when there are less than 1000 tax returns" do
+          it "shows the full amount of tax returns" do
+            get :index, params: params
 
-          expect(assigns(:tax_return_count)).to eq 50
+            expect(assigns(:tax_return_count)).to eq 50
```

Comment:
Presumably the right hand side of the equality check needs the same `.to_s` that the implementation got!